common/changes/office-ui-fabric-react/edwl-09-2018-removeOUFRTslintOverrides_2018-09-14-19-11.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Remove max lines TS override",
+      "type": "patch"
```

Comment:
This should be none, since the API surface did not change (neither did code behavior)